fx-src/index.js

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+"use strict";
+
+const Cu = Components.utils;
+
+Cu.importGlobalProperties(['fetch']);
+const { EventEmitter } = Cu.import("resource://devtools/shared/event-emitter.js", {});
+
+export default class KintoHttpClient extends KintoClient {
+  constructor(remote, options={}) {
+    const emitter = {};
+    EventEmitter.decorate(emitter);
+    super({...options, events: emitter});
```

Comment:
We shouldn't override `events` by default here, as it's possibly been instantiated already and passed to the constructor, and we probably want to preserve it.